unicodetools/data/ucd/dev/UnicodeData.txt

```diff
@@ -31858,6 +31858,971 @@ FFFD;REPLACEMENT CHARACTER;So;0;ON;;;;;N;;;;;
 18DF0;TANGUT COMPONENT-881;Lo;0;L;;;;;N;;;;;
 18DF1;TANGUT COMPONENT-882;Lo;0;L;;;;;N;;;;;
 18DF2;TANGUT COMPONENT-883;Lo;0;L;;;;;N;;;;;
+18E00;JURCHEN CHARACTER-18E00;Lo;0;L;;;;;N;;;;;
```

Comment:
If these names are deliberately algorithmic, should we use the range/start..end notation in UnicodeData.txt?

@Ken-Whistler

Reply:
We didn’t for Egyptian, and there are three times fewer here than for Egyptian, so I’d rather not (I did it for Seal and it is somewhat annoying).